spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/TableWriter.scala

```diff
@@ -79,8 +95,11 @@ class TableWriter[T] private (
   private val isCounterUpdate =
     tableDef.columns.exists(_.isCounterColumn)
 
+  private val containsCollectionBehaviors =
+    columnSelector.collect{ case cn:CollectionColumnName => cn }.nonEmpty
```

Comment:
This is one of those times where i'm not sure if I should be using isInstanceOf. Do we have a style guide where we allow using the method rather than doing type matching?